### DISCUSSION TITLE
WinMerge android app

### DISCUSSION THREAD

**Discussion by ctrl8511** [WinMerge/winmerge#925] [General]
Goodmorning. 

There is a way in the future for an android version of WinMerge? 
Or have you some android apps to suggest? To compare 2 files on android :)

I already tried with "DiffFinder" and "DiffTool" and "Hash And Compare" 
but they don't work, how can compare 2 files on android?

Best regards.

**Comment by sdottaka** [comment]
There are no plans to make WinMerge for Android.

You may want to try a diff tool available in your web browser, such as

https://www.diffnow.com/compare-files
https://www.diffchecker.com/
https://editor.mergely.com/

**Comment by Werve** [comment]
For those who were looking, I would like to share the only app I found that has a similar role, to compare files and folders. 
https://play.google.com/store/apps/details?id=com.difftool

**Comment by Max-RM** [comment]
> Goodmorning.
> 
> There is a way in the future for an android version of WinMerge? Or have you some android apps to suggest? To compare 2 files on android :)
> 
> I already tried with "DiffFinder" and "DiffTool" and "Hash And Compare" but they don't work, how can compare 2 files on android?
> 
> Best regards.

![Screenshot_2023-04-23-20-39-44-495_com eltechs ed](https://user-images.githubusercontent.com/81485476/233850078-60d4b1b4-ed3f-4d70-af93-c0cea08f9de3.jpg)
![Screenshot_2023-04-23-20-39-48-129_com eltechs ed](https://user-images.githubusercontent.com/81485476/233850083-60accb17-40ed-49e4-b77b-5403335789dd.jpg)
![Screenshot_2023-04-23-20-40-53-024_com eltechs ed](https://user-images.githubusercontent.com/81485476/233850085-0fde0874-c41c-47be-be05-3430a6e57e6a.jpg)
![Screenshot_2023-04-23-20-41-04-023_com eltechs ed](https://user-images.githubusercontent.com/81485476/233850087-0f9c518c-5501-4c79-8966-2aee6add4961.jpg)
![Screenshot_2023-04-23-20-41-08-485_com eltechs ed](https://user-images.githubusercontent.com/81485476/233850089-ea165bf6-a48e-4ed2-bbc7-2b18cd820aae.jpg)
![Screenshot_2023-04-23-20-39-06-360_com eltechs ed](https://user-images.githubusercontent.com/81485476/233850092-0fe1c363-365d-41db-95e0-732915b09ca9.jpg)

**Reply by salmanmemon7** [reply]
How on Android phone

**Comment by Max-RM** [comment]
Run on Android via Exagear. Thx devs for support x86, because Exagear can't run x64 apps

**Comment by Max-RM** [comment]
But yea. Native port will be better. Or at latest support Linux arm. Because then we can run such program via Termux or Linux Deploy with native performance. And it's easy to mount Android folders into Linux in container.
